FutureNorthants/northants-website
# Add Image slice

## This PR relies on the following:
- https://github.com/FutureNorthants/northants-website/pull/279
- https://github.com/FutureNorthants/northants-design-system/pull/112

## To Test
- Checkout the design system branch, then run yalc publish
- Checkout the backend PR and import config
- Checkout this branch on the frontend and run yalc add northants-design-system && yarn install && yarn dev
- In a new window, run yarn codegen to ensure there are no changes to files
- Test a page where an image has been added in the body and in an accordion
- Ensure the image is the correct image ratio

**Note**: I may wait for this [PR](https://github.com/FutureNorthants/northants-website/pull/269) and then refactor the graphql
